app/scripts/controllers/transactions/index.js

```diff
@@ -1161,13 +1161,24 @@ export default class TransactionController extends EventEmitter {
    *  params instead of allowing this method to generate them
    * @param options
    * @param options.estimatedBaseFee
+   * @param actionId
    * @returns {txMeta}
    */
   async createCancelTransaction(
     originalTxId,
     customGasSettings,
     { estimatedBaseFee } = {},
+    actionId,
```

Comment:
I think the same feedback that @digiwand left on the other pr applies here, IE this should either be a part of the options bag object, or moved to ahead of the options bag.

Reply:
I updated the PR to implement feedback.